docs/_rpc/ns-eth.md

```diff
@@ -184,3 +184,60 @@ And the result is the Ethereum ABI encoded threshold number:
 ```
 
 Just for the sake of completeness, decoded the response is: `2`.
+
+### eth_createAccessList
+
+This method creates an [EIP2930](https://eips.ethereum.org/EIPS/eip-2930) type `accessList` based on a given `Transaction`.
+
+If your `transaction` has code executed, then you can generate `transaction` `accessList` with `eth_createAccessList`. 
+If you send it with your `Transaction` then it will lower your gas cost on Ethereum
+
+#### Parameters
+
+| Field              | Type       | Description          |
+|:-------------------|:-----------|:---------------------|
+| `transactionCall`  | `Object`   | `Transaction` object |
+| `blockParameter`   | `Object`   | Optional             |
+| `optimize`         | `Boolean`  | Optional             |
+
+#### Usage
+
+```
+curl --data '{"method":"eth_createAccessList","params":[transactionCall, blockParameter, optimize],"id":1,"jsonrpc":"2.0"}' -H "Content-Type: application/json" -X POST localhost:8545
```

Comment:
The `data` here isn't valid json, that can't possibly work.. ?

Reply:
Appreciate that, bu

> The `data` here isn't valid json, that can't possibly work.. ?

I am not really sure how we can demonstrate a working example without bloating the example. Perhaps I can indicate that the params are to be populated by the user? 

It can be assumed that the function of this is in the context of a transaction, so it wouldn't be incorrect to assume that the direction of the developer will understand that they need to populate the params.